apps/ledger-live-desktop/src/newArch/features/ModularDrawer/screens/AccountsWarning/components/ActionButtons.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Button, Flex } from "@ledgerhq/react-ui";
+import { ActionButtonsProps } from "../types";
+
+export const ActionButtons = ({ action1, action2 }: ActionButtonsProps) => {
```

Comment:
```suggestion
export const ActionButtons = ({ primaryAction,
  secondaryAction }: ActionButtonsProps) => {
```
Maybe more accurate ?